sandbox/plc4c/generated-sources/modbus/src/modbus_constants.c

```diff
@@ -24,18 +24,18 @@
 #include "modbus_constants.h"
 
 // Parse function.
-plc4c_return_code plc4c_modbus_read_write_modbus_constants_parse(plc4c_spi_read_buffer* buf, plc4c_modbus_read_write_modbus_constants** message) {
+plc4c_return_code plc4c_modbus_read_write_modbus_constants_parse(plc4c_spi_read_buffer* buf, plc4c_modbus_read_write_modbus_constants** _message) {
```

Comment:
to me, when I see _var_name I think of a private var, not only in python but in some c-ish languages I've worked in.  is there some standard that you are writing two using _ in a parameter name?

Reply:
Well I had to somehow make sure the names a user can choose in mspec don't interfere with those used internally ... so I prefixed them with a "_" if you have another suggestion, I'm  fine with changing.